app/src/pages/admin/serverSettingsPage/serverSettingsTabs/authConfigurationTab/forms/ssoUsersForm/ssoUsersForm.scss

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+:global {
+  .custom-text-wrapper {
+    width: 100%;
+    max-width: 700px;
+    padding-left: 0;
+  }
+
+  .form-field {
+    align-items: flex-start;
+  }
+
+  .custom-span-class {
+    display: block;
+    font-size: 12px;
+    line-height: 1.5;
+    color: $COLOR--gray-60;
+    margin-left: 9px;
+    width: 300px;
+  }
+
+  .enabled-description {
+    margin-top: 0;
+  }
+
```

Comment:
Please also adjust the description position as discussed in MS teams.

Reply:
Done.